src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -427,12 +335,10 @@ MavlinkReceiver::send_flight_information()
 {
 	mavlink_flight_information_t flight_info{};
 
-	param_t param_flight_uuid = param_find("COM_FLIGHT_UUID");
+	uint64_t flight_uuid = static_cast<uint64_t>(_p_flight_uuid.get());
 
-	if (param_flight_uuid != PARAM_INVALID) {
-		int32_t flight_uuid;
-		param_get(param_flight_uuid, &flight_uuid);
-		flight_info.flight_uuid = (uint64_t)flight_uuid;
+	if (flight_uuid != PARAM_INVALID) {
```

Comment:
It can't be invalid anymore.

Reply:
`PARAM_INVALID` is a `uint16_t,` so if  it's evaluated `!=` against a `uint64_t` that should have still worked, right? (I think?)

Reply:
Eek, you're right.  I'll fix that right up.